### PR TITLE
pulseaudio post 16.0 fixes

### PR DESCRIPTION
**Pull request by i-garrison**
media-libs/libpulse: Fix parsing of percentages with decimal points
media-sound/pulseaudio-daemon: Add optional conf dirs to silence warning https://bugs.gentoo.org/852587
media-sound/pulseaudio-daemon: Fix crash after combine-sink underrun https://bugs.gentoo.org/852848
media-sound/pulseaudio-daemon: Fix gstreamer bluetooth crash on 32-bit ARM
media-sound/pulseaudio-daemon: Fix crash using pacmd play-file command

### CODE REVIEW

File: media-libs/libpulse/files/pulseaudio-16.0-fix-pactl-volume-command.patch [hunk 1]
```diff
@@ -0,0 +1,53 @@
+From 05c06afa58e30b7958e96766d1e917099c8a4041 Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline media-libs/libpulse/files/pulseaudio-16.0-fix-pactl-volume-command.patch:1]: Nit: Could you add a link to the relevant commit at the top of each patch?

**Comment by thesamesam** [comment]
Also, I'd consider not `git mv`ing for this, just in case (`cp` instead).

Nice work and thanks for doing this so quickly!

File: media-libs/libpulse/files/pulseaudio-16.0-fix-pactl-volume-command.patch (hunk 1 above)
Comment (i-garrison) [reply media-libs/libpulse/files/pulseaudio-16.0-fix-pactl-volume-command.patch:3]: done now!

**Comment by thesamesam** [comment]
Great work, thank you!

**Pull request closed without merging**